vendor/tekzitel/tekzipark-profile-868.yaml

```diff
@@ -0,0 +1,52 @@
+# Vendor profile ID, can be freely issued by the vendor
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
+vendorProfileID: 42
```

Comment:
This doesn't seem right.

Leave empty if you don't use one.

Reply:
Yes, will add the examples and comment out the vendorProfileID